src/main/java/com/onlinebookstore/onlinebookstore/model/ShoppingCart.java

```diff
@@ -0,0 +1,29 @@
+package com.onlinebookstore.onlinebookstore.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.util.Set;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = "shoppingcarts")
+@Getter
+@Setter
+public class ShoppingCart {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne
+    @JoinColumn(name = "user_id", nullable = false)
+    private User user;

```

Comment:
would be nice to have use user_id as a primary key of the cart, because we have a 1-1 relation here, use `@MapsId`for that. But that is not too important